src/tags_int.cpp

```diff
@@ -833,7 +833,13 @@ namespace Exiv2 {
                 N_("Contains four ASCII characters representing the TIFF/EP standard "
                 "version of a TIFF/EP file, eg '1', '0', '0', '0'"),
                 ifd0Id, tiffEp, unsignedByte, 4, printValue), // TIFF/EP Tag
-        TagInfo(0x9217, "SensingMethod", N_("Sensing Method"), N_("Type of image sensor."), ifd0Id, tiffEp, unsignedShort, 1, printValue), // TIFF/EP tag
+        TagInfo(0x9217, "SensingMethod", N_("Sensing Method"), N_("Type of image sensor."), exifId, tiffEp, unsignedShort, 1, printValue), // TIFF/EP tag
```

Comment:
0x9217 should stay associated w/ ifd0Id, this one is TIFF/EP specific. There is a separate (and equivalent) 0xa217 for the Exif IFD.

But this one should probably also use print0xa217.

Reply:
OK, let's use print0xa217().